### ISSUE TITLE
Possible bug in mutant unparser

### ISSUE THREAD

**Issue by nepalez** [mbj/mutant#434]
Hi!

Seems like there is a bug in mutant unparser.

It takes the method

``` ruby
# /home/nepalez/ruby/rom/rom-cassandra/lib/rom/cassandra/migrations/generator.rb
def source
  File.expand_path("../generator/migration.erb", __FILE__)
end
```

The result is expected to be `../rom/cassandra/migrations/generator/migration.erb` (and spec passes).

But mutant unparses this it into:

```
--- Neutral failure ---
Original code was inserted unmutated. And the test did NOT PASS.
Your tests do not pass initially or you found a bug in mutant / unparser.
Subject AST:
(def :source
  (args)
  (send
    (const nil :File) :expand_path
    (str "../generator/migration.erb")
    (str "(string)")))
Unparsed Source:
def source
  File.expand_path("../generator/migration.erb", "(string)")
end
Test Result:
- 1 @ runtime: 0.145323166
  - rspec:69:./spec/unit/migrations/generator_spec.rb:18/ROM::Cassandra::Migrations::Generator.call creates the migration
```

With error:

```
Failures:

  1) ROM::Cassandra::Migrations::Generator.call creates the migration
     Failure/Error: subject { described_class.call(name, path) }
     Errno::ENOENT:
       No such file or directory @ rb_sysopen - /home/nepalez/ruby/rom/rom-cassandra/generator/migration.erb
```

As you can see its works as if instead of:

``` ruby
File.expand_path("../generator/migration.erb", __FILE__)
```

there were:

``` ruby
File.expand_path("../generator/migration.erb")
```

**Comment by mbj** [comment]
Thx for the report. All hail to noop/neutral mutation self tests!

**Comment by mbj** [comment]
@nepalez This was fixed with mutant-0.8.21 but I forgot to close it (and add specs). The fix was a side effect to move to the modern parser AST format.
